src/main/java/org/spongepowered/api/map/MapScales.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.map;
+
+import org.spongepowered.api.util.generator.dummy.DummyObjectProvider;
+
+/**
+ * An enumeration of all the possible vanilla {@link MapScales}s.
+ */
+public final class MapScales {
+
+    //SORTFIELDS:ON
+
+    /**
+     * Represents a 4:1 block to pixel ratio scaling.
+     */
+    public static final MapScale SCALE_1 = DummyObjectProvider.createFor(MapScale.class, "SCALE_1");
```

Comment:
Doesn't scale to chunk size, at that point it becomes less clear, that was
an initial naming I used and it was very confusing.
On Apr 4, 2016 5:21 PM, "Simon816" <notifications@github.com> wrote:

> In src/main/java/org/spongepowered/api/map/MapScales.java
> <https://github.com/SpongePowered/SpongeAPI/pull/1162#discussion_r58459287>
> :
>
> > + */
> > +package org.spongepowered.api.map;
> > +
> > +import org.spongepowered.api.util.generator.dummy.DummyObjectProvider;
> > +
> > +/**
> > + * An enumeration of all the possible vanilla {@link MapScales}s.
> > + */
> > +public final class MapScales {
> > +
> > +    //SORTFIELDS:ON
> > +
> > +    /**
> > +     * Represents a 4:1 block to pixel ratio scaling.
> > +     */
> > +    public static final MapScale SCALE_1 = DummyObjectProvider.createFor(MapScale.class, "SCALE_1");
>
> What about naming these SCALE_4, SCALE_16 etc so I don't need to look up
> the javadocs to find what scaling this is?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/SpongePowered/SpongeAPI/pull/1162/files/d8338269be828e278d15ed5ffbe9bb30b7c899c1#r58459287>
>